README.rst

```diff
@@ -0,0 +1,260 @@
+=====================
+PyTorch Image Quality
+=====================
+
+.. image:: https://user-images.githubusercontent.com/22414094/106116361-df8a3c80-6162-11eb-83df-f5838110e614.png
+
+..
+
+  PyTorch Image Quality (PIQ) is not endorsed by Facebook, Inc.;
+
+  PyTorch, the PyTorch logo and any related marks are trademarks of Facebook, Inc.
+
+
+.. image:: https://badge.fury.io/py/piq.svg
+   :target: https://badge.fury.io/py/piq
+   :alt: Pypi Version
+.. image:: https://anaconda.org/photosynthesis-team/piq/badges/version.svg
+   :target: https://anaconda.org/photosynthesis-team/piq
+   :alt: Conda Version
+.. image:: https://github.com/photosynthesis-team/piq/workflows/flake-8%20style%20check/badge.svg
+   :alt: CI flake-8 style check
+.. image:: https://github.com/photosynthesis-team/piq/workflows/testing/badge.svg
+   :alt: CI testing
+.. image:: https://codecov.io/gh/photosynthesis-team/piq/branch/master/graph/badge.svg
+   :target: https://codecov.io/gh/photosynthesis-team/piq
+   :alt: codecov
+.. image:: https://sonarcloud.io/api/project_badges/measure?project=photosynthesis-team_photosynthesis.metrics&metric=alert_status
+   :target: https://sonarcloud.io/dashboard?id=photosynthesis-team_photosynthesis.metrics
+   :alt: Quality Gate Status
+
+
+
+`PyTorch Image Quality (PIQ) <https://github.com/photosynthesis-team/piq>`_ is a collection of measures and metrics for 
+image quality assessment. PIQ helps you to concentrate on your experiments without the boilerplate code.
+The library contains a set of measures and metrics that is continually getting extended.
+For measures/metrics that can be used as loss functions, corresponding PyTorch modules are implemented.
+
+We provide:
+
+* Unified interface, which is easy to use and extend.
+* Written on pure PyTorch with bare minima of additional dependencies.
+* Extensive user input validation. You code will not crash in the middle of the training.
+* Fast (GPU computations available) and reliable.
+* Most metrics can be backpropagated for model optimization.
+* Supports python 3.6-3.8.
+
+PIQ was initially named `PhotoSynthesis.Metrics <https://pypi.org/project/photosynthesis-metrics/0.4.0/>`_
+
+.. contents:: Table of Contents
```

Comment:
I suggest to use heading for table go content. It is lost in plain text.
![image](https://user-images.githubusercontent.com/22414094/107883789-181b6b80-6f02-11eb-92a3-5e3cf376eb75.png)